### DISCUSSION TITLE
InvocableMethod annotation support

### DISCUSSION THREAD

**Discussion by LuisPerezCamacho** [forcedotcom/salesforcedx-vscode#5815] [Ideas]
Is your feature request related to a problem? Please describe.
It would be great if you could add autocomplete options for the InvocableMethod annotation. The annotation supports 5 modifiers described [here](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_classes_annotation_InvocableMethod.htm).

Describe the solution you'd like
I would like to see a dropdown list with options for the 5 optional modifiers for the annotation InvocableMethod when I write the annotation.

Describe alternatives you've considered
There isn't a clear alternative to this feature that I can think of.

Additional context

**Comment by git2gus** [comment]
This issue has been linked to a new work item: W-13465509
